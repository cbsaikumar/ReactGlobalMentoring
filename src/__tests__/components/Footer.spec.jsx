import Footer from '../../components/Footer';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Footer component', () => {
    it('should render correctly', () => {
        const tree = renderer
            .create(<Footer />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})