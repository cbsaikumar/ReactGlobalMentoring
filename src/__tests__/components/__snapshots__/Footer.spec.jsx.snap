// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<footer
  className="bar"
>
  <h5
    className="red-text text-darken-2 grey darken-3 grey-bar"
  >
    netflixroulette
  </h5>
</footer>
`;
